lib/platform/common.ts

```diff
@@ -93,15 +98,27 @@ export interface RepoParams {
 }
 
 /**
- * TODO: Proper typing
+ *
  */
-export type Pr = {
+export interface Pr {
+  body?: string;
   branchName: string;
```

Comment:
Maybe the Azure naming is useful here? `sourceBranch` and `targetBranch`?

Reply:
Yes, would do the renaming in separate pr, because this will change a lot code